src/apps/backend/modules/booking/booking-service.ts

```diff
@@ -0,0 +1,21 @@
+import { Booking, CreateBookingRequestParams } from './types';
+
+export default class BookingService {
+  static async createBooking({
+    firstName,
+    phoneNumber,
+    address,
+    schedule,
+  }: CreateBookingRequestParams): Promise<Booking> {
+    if (!firstName || !phoneNumber || !address) {
+      throw new Error('Customer details are required');
```

Comment:
Can you please Google `How to write good error message` and change this accordingly?